src/pointer.rs

```diff
@@ -1930,7 +2085,7 @@ mod tests {
 
     #[quickcheck]
     fn qc_pop_and_push(mut ptr: PointerBuf) -> bool {
-        let original_ptr = ptr.clone();
+        let subjectal_ptr = ptr.clone();
```

Comment:
I think this could be just `subject_ptr` (I'm not sure 'subjectal' is a recognised word, but if it is it's pretty archaic).